### PR TITLE
Fix RethinkDB installation on Travis-ci

### PR DESCRIPTION
**Pull request by christiankakesa**

### CODE REVIEW

File: spec/reql_spec.cr [hunk 1]
```diff
@@ -4,27 +4,37 @@ r.db("test").table_list.for_each do |name|
...
+  # {{ run("./reql_spec_generator", "spec/rql_test/src/math_logic/div.yaml") }}
+  # {{ run("./reql_spec_generator", "spec/rql_test/src/math_logic/floor_ceil_round.yaml") }}
+  # {{ run("./reql_spec_generator", "spec/rql_test/src/math_logic/logic.yaml") }}
+  # {{ run("./reql_spec_generator", "spec/rql_test/src/math_logic/math.yaml") }}
+  # {{ run("./reql_spec_generator", "spec/rql_test/src/math_logic/mod.yaml") }}
+  # {{ run("./reql_spec_generator", "spec/rql_test/src/math_logic/mul.yaml") }}
+  # {{ run("./reql_spec_generator", "spec/rql_test/src/math_logic/sub.yaml") }}
+  # {{ run("./reql_spec_generator", "spec/rql_test/src/mutation/update.yaml") }}
+  # {{ run("./reql_spec_generator", "spec/rql_test/src/range.yaml") }}
+  # {{ run("./reql_spec_generator", "spec/rql_test/src/selection.yaml") }}
```
Comment (caspiano) [inline spec/reql_spec.cr:39]: Any reason for commenting these tests out?

**Review by caspiano — COMMENTED** [review]
Hi @fenicks, thanks for the PR!
Could you squash this into a single commit that only contains the CI changes?

**Comment by christiankakesa** [comment]
> Hi @fenicks, thanks for the PR!
> Could you squash this into a single commit that only contains the CI changes?

Of course. I'll do that.

File: spec/reql_spec.cr (hunk 1 above)
Comment (christiankakesa) [reply spec/reql_spec.cr:39]: Not needed in this PR.
I start ading more features not handled yet in the drivers but handled in tests.
I'll revert that for this PR.

**Comment by caspiano** [comment]
> > Hi @fenicks, thanks for the PR!
> > Could you squash this into a single commit that only contains the CI changes?
> 
> Of course. I'll do that.

I'll cherry-pick the travis commit onto master

**Comment by caspiano** [comment]
Merged `Fix Rethinkdb installation on Travis-ci` to master (4dfc20e)

**Pull request closed without merging**